src/styles/index.scss

```diff
@@ -1,3 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+:root {
+  --main-background-color: #fff;
+  --border-color: #f3f3f3;
+  --button-border-color: #00acdc;
+}
+
+.product-card {
+  width: 200px;
+  background-color: var(--main-background-color);
+  border-radius: 5px;
+  border: 1px solid var(--border-color);
+}
+
+.product-card__title {
```

Comment:
pls make use of [sass nesting](https://css-tricks.com/using-sass-control-scope-bem-naming/) for all selectors in the file